### PR TITLE
feat: added LogoComponent and Integrated with Login and Signup pages

### PR DESCRIPTION
**Pull request by MrBaileyisme**
### Description
- Added TreeTraderLogo.png to public assets
- Created a basic LogoComponent
- Added LogoComponent to Login and Signup pages

**Resolves**: #259

---

### Changes Made
- [ ] Changes in **`apps`** folder (specify the app and briefly describe the changes):
  - [x] `Web`
  - [ ] `Native`

- [ ] Changes in **`packages`** folder (specify the package and briefly describe the changes):
  - [ ] `Core`

---

### Type of Change
- [ ] 🐛 **Bug fix** (non-breaking change which fixes an issue)
- [x] ✨ **New feature** (non-breaking change which adds functionality)
- [ ] 💥 **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 **Documentation update** (changes)

---

## Screenshots
|       Before        |       After        |
| :-----------------: | :----------------: |
| image | Screenshot 2024-11-14 223256
 |
---

### How Has This Been Tested?
- [ ] Cypress integration
- [ ] Cypress component tests
- [ ] Jest unit tests

---

### Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

---

### Additional Comments
*(Optional) Add any additional comments or notes for reviewers here.*

### CODE REVIEW

File: apps/web/src/app/login/page.tsx
Comment (dorukucak) [inline apps/web/src/app/login/page.tsx:1]: Think there is a typo here. LoginComponent vs LogoComponent

Comment (MrBaileyisme) [reply apps/web/src/app/login/page.tsx:1]: Lol sorry. Will fix that in the morning

File: apps/web/public/assets/images/TreeTraderLogo.png
Comment (pierrelstan) [inline apps/web/public/assets/images/TreeTraderLogo.png:1]: SVG  format is a better choice for logos due to its scalability, smaller file size, and flexibility.

Comment (MrBaileyisme) [reply apps/web/public/assets/images/TreeTraderLogo.png:1]: Done

File: apps/web/src/app/signup/page.tsx [hunk 1]
```diff
@@ -99,6 +100,7 @@ export default function SignUp() {
   return (
     <Wrapper>
       <CenteredColumnBox>
+        <LogoComponent />
```
Comment (pierrelstan) [inline apps/web/src/app/signup/page.tsx:103]: Hey, I suggest renaming LogoComponent to Logo. It’s more concise and easier to understand while maintaining consistency with other component names.

Comment (MrBaileyisme) [reply apps/web/src/app/signup/page.tsx]: Done!

**Comment by pierrelstan** [comment]
@MrBaileyisme Thank you for your work!

**Pull request merged**